src/wrappers/_MarkdownWrapper.jsx

```diff
@@ -0,0 +1,71 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import TerraMarkdown from 'terra-markdown';
+import classNames from 'classnames/bind';
+import ContentLoaded from './_ContentLoaded';
+import ContentLoading from './_ContentLoading';
+import ErrorPage from './_ErrorPage';
+
+import styles from './Markdown.module.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * The content to be placed within the main content area of the container.
+   */
+  content: PropTypes.func,
+};
+
+const defaultProps = {
+  content: undefined,
+};
+
+const MarkdownWrapper = ({ content }) => {
+  const [state, setState] = useState({
+    markdown: undefined,
+    isErrored: false,
+  });
+  const { markdown, isErrored } = state;
+
+  // Dynamically import Javascript chunk
+  useEffect(() => {
+    let isActive = true;
+    if (!markdown && !isErrored) {
+      content().then(({ default: markdownFile }) => {
```

Comment:
Should content be required or should we check to ensure content is defined? I'm assuming this isn't likely an issue because this is generated with the dev-site config script